test/ttt_test.exs

```diff
@@ -3,38 +3,56 @@ defmodule TTTTest do
   import ExUnit.CaptureIO
   doctest TTT
 
-  test "Game will automatically switch turn to 'o' after 'x' has played and vice versa." do
+  test "Game will automatically switch turn to 'o' after 'x' has played and vice versa" do
     first_player = :x
     assert TTT.switch_turn(first_player) == :o
 
     first_player = :o
     assert TTT.switch_turn(first_player) == :x
   end
 
-  test "Only a string of '1' to '9' will be accepted as valid input." do
-    input1 = "1"
-    input2 = " "
-    input3 = "aa"
-    input4 = ",,"
-    input5 = "10"
-    input6 = "0"
-    input7 = "5"
-    assert TTT.valid_input?(input1) == :true
-    assert TTT.valid_input?(input2) == :false
-    assert TTT.valid_input?(input3) == :false
-    assert TTT.valid_input?(input4) == :false
-    assert TTT.valid_input?(input5) == :false
-    assert TTT.valid_input?(input6) == :false
-    assert TTT.valid_input?(input7) == :true
+  test "A number '1' through '9' will be a valid input" do
+    inputs = ["1", "2", "5", "8", "9"]
+    for input <- inputs, do: assert TTT.valid_input?(input) == true
+  end
+
+  test "Whitespaces or newlines before or/and after a valid input will be ignored, such input will be a valid one" do
```

Comment:
This should be split into at least two or three tests - whitespace before, whitespace after, newlines after, etc